src/Nox.Solution/Models/Presentation/ApiConfiguration/ApiRouteMappings/ApiRouteMapping.cs

```diff
@@ -47,12 +47,10 @@ public class ApiRouteMapping : YamlConfigNode<NoxSolution, ApiConfiguration>
     public ContentBodyType RequestBodyType { get; internal set; } = ContentBodyType.None;
 
     [IfEquals(nameof(RequestBodyType), ContentBodyType.Json)]
-    [Required]
     public JsonTypeDefinition JsonBodyType { get; internal set; } = null!;
```

Comment:
make nullable explicit in such cases
```suggestion
    public JsonTypeDefinition? JsonBodyType { get; internal set; } = null!;
```